.github/workflows/documentation.yml

```diff
@@ -103,17 +139,193 @@ jobs:
         with:
           name: dnsdist-html-docs-${{steps.get-version.outputs.pdns_version}}
           path: ./pdns/dnsdistdist/docs/_build/dnsdist-html-docs.tar
-      - run: bzip2 dnsdist-html-docs.tar
-        if: ${{github.ref_name == 'master'}}
-        working-directory: ./pdns/dnsdistdist/docs/_build
       - run: inv ci-docs-build-pdf
         working-directory: ./pdns/dnsdistdist
       - uses: actions/upload-artifact@v4
         with:
           name: dnsdist-${{steps.get-version.outputs.pdns_version}}.pdf
           path: ./pdns/dnsdistdist/docs/_build/latex/dnsdist.pdf
-      - run: inv ci-docs-upload-master --docs-host="${DOCS_HOST}" --pdf="dnsdist.pdf" --username="dnsdist_org" --product="dnsdist"
+      - run: docker logs ${{ steps.proxpi-docker.outputs.id }} 2>&1 | grep whl | awk '{print $8}' | cut -d "/" -f 4 | awk -F'-' '{print $1"=="$2}' | sort -u --ignore-case > /tmp/proxpi-dnsdist.log
+      - id: pip-logs-dnsdist
+        run: echo "list-dnsdist=$(cat /tmp/proxpi-dnsdist.log | base64 -w0)" >> "$GITHUB_OUTPUT"
+      - name: Copy PDF to HTML docs directory
+        run: |
+          cp ./pdns/dnsdistdist/docs/_build/latex/dnsdist.pdf ./pdns/dnsdistdist/docs/_build/dnsdist-html-docs/
+
+      # Build website
+      - name: Build website
+        run: |
+          tar cf website.tar website
+      - uses: actions/upload-artifact@v4
+        with:
+          name: website-${{steps.get-version.outputs.pdns_version}}
+          path: website.tar
+
+  publish-to-netlify:
+    name: Publish to Netlify
+    needs: build-docs
+    if: ${{ vars.PUBLISH_DOCS_TO_NETLIFY == 'true' && github.ref_name == 'master' }}
+    runs-on: ubuntu-22.04
+    env:
+      NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_API_TOKEN }}
+      NETLIFY_SITE_ID_DOCS: ${{ vars.NETLIFY_SITE_ID_DOCS }}
+      NETLIFY_SITE_ID_DNSDIST: ${{ vars.NETLIFY_SITE_ID_DNSDIST }}
+    steps:
+      - name: Check required secrets
+        run: |
+          missing_secrets=()
+          [ -z "$NETLIFY_AUTH_TOKEN" ] && missing_secrets+=("NETLIFY_API_TOKEN")
+          [ -z "$NETLIFY_SITE_ID_DOCS" ] && missing_secrets+=("NETLIFY_SITE_ID_DOCS")
+          [ -z "$NETLIFY_SITE_ID_DNSDIST" ] && missing_secrets+=("NETLIFY_SITE_ID_DNSDIST")
+          if [ ${#missing_secrets[@]} -ne 0 ]; then
+            echo "Error: The following secrets are missing: ${missing_secrets[*]}"
+            exit 1
```

Comment:
Will this result in me getting a :x: in my fork? that'd be lousy.

It's much better to just skip the remaining steps and treat it as a success.